CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## UNRELEASED
+
+### Adds
+
+* Add translation keys used by the multisite pro module.
+* Add the possibility to set dynamic values in the fields' `help` and `htmlHelp` attributes following this pattern: `translationKey|foo=bar|fizz=buzz`,
```

Comment:
cf https://github.com/apostrophecms/apostrophe/pull/4437/files#r1502556802

will need a cypress test if we continue with this solution